README.md

```diff
@@ -159,27 +153,27 @@ and tooling developers can use to test their implementations.
 
 Currently `cargo component` targets `wasm32-wasi` by default.
 
-As this target is for a _preview1_ release of WASI, the WebAssembly module 
-produced by the Rust compiler must be adapted to the _preview2_ version of WASI 
+As this target is for a _preview1_ release of WASI, the WebAssembly module
+produced by the Rust compiler must be adapted to the _preview2_ version of WASI
 supported by the component model.
 
 The adaptation is automatically performed when `wasm32-wasi` is targeted.
 
-To prevent this, override the target to `wasm32-unknown-unknown` using the 
+To prevent this, override the target to `wasm32-unknown-unknown` using the
 `--target` option when building. This, however, will disable WASI support.
 
-Use the _preview2_ version of [`wasi-common`][2] in your host to run components 
+Use the _preview2_ version of [`wasi-common`][2] in your host to run components
 produced by `cargo component`.
 
-When the Rust compiler supports a [_preview2_ version of the WASI target][1], 
+When the Rust compiler supports a [_preview2_ version of the WASI target][1],
 support in `cargo component` for adapting a _preview1_ module will be removed.
 
 [1]: https://github.com/rust-lang/compiler-team/issues/594
 [2]: https://github.com/bytecodealliance/preview2-prototyping/tree/main/wasi-common
 
 ## Getting Started
 
-Use `cargo component new --lib <name>` to create a new reactor component.
+Use `cargo component new --reactor <name>` to create a new reactor component.
```

Comment:
If we're going to say "reactor" here, it'd be good to have a brief explanation of what that means. Perhaps, "... which is a component that doesn't have a `run` function, and is meant to be used as a library".